backend/organization/models/members.py

```diff
@@ -114,3 +114,76 @@ class Meta:
 
     def __str__(self):
         return "User %d member for organization %s" % (self.user.id, self.organization.name)
+
+class MembershipRequests(models.Model):
+
+    id = models.AutoField(help_text='identifier of a join request'
+                                , primary_key=True
+                                ,verbose_name="join request Id")
```

Comment:
Non-blocker: Format this like `user` column on line 125

Reply:
You don't have to identify `id` since django automatically identify this.

Reply:
Yes to formatting. But `id` is auto generated field in django so you don't really need to identify it in the code.

Reply:
Handled in 5963944